module/VuFind/tests/unit-tests/src/VuFindTest/OAuth2/Repository/AbstractTokenRepositoryTestCase.php

```diff
@@ -261,7 +219,7 @@ protected function findAccessTokenTableRow(array $data): ?int
      *
      * @return MockObject&AccessTokenServiceInterface
      */
-    protected function getMockAccessTokenService(): AccessTokenServiceInterface
+    protected function getMockAccessTokenService(): AccessTokenServiceInterface|MockObject
```

Comment:
I think this should be an "and" rather than an "or":

```suggestion
    protected function getMockAccessTokenService(): AccessTokenServiceInterface&MockObject
```